src/main/java/com/g4s8/ghman/utility/EnvironmentVariables.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.utility;
+
+import java.util.Map;
+import org.cactoos.text.FormattedText;
+
+/**
+ * Environment Variables Retriever.
+ *
+ * @since 1.0
+ */
+public final class EnvironmentVariables {
+
+    /**
+     * Github Client Id Key String.
+     */
+    private static final String GH_CLIENT = "GH_CLIENT";
+
+    /**
+     * Github Secret Key String.
+     */
+    private static final String GH_SECRET = "GH_SECRET";
+
+    /**
+     * Application Host Key String.
+     */
+    private static final String APP_HOST = "APP_HOST";
+
+    /**
+     * Environment Variables Map.
+     */
+    private final Map<String, String> envvars;
+
+    /**
+    * Ctor.
+    */
+    public EnvironmentVariables() {
+        this(System.getenv());
+    }
+
+    /**
+    * Ctor.
+    * @param envvars Preloaded Environment Variables Map
+    */
+    public EnvironmentVariables(final Map<String, String> envvars) {
+        this.envvars = envvars;
+    }
+
+    /**
+     * Retrieve GitHub Client Id.
+     * @return Github Client Id
+     * @throws IllegalStateException if Github Client Id is missing
+     *  from system environment variables.
+     */
+    public String getGithubClientId() throws IllegalStateException {
+        return this.envvars.computeIfAbsent(
+            this.GH_CLIENT,
+            key -> {
+                this.missingKeyExceptionThrower(key);
+                return null;
+            }
+        );
+    }
+
+    /**
+     * Retrieve GitHub Client Secret.
+     * @return Github secret
+     * @throws IllegalStateException if Github Secret is missing from
+     *  system environment variables.
+     */
+    public String getGithubClientSecret() throws IllegalStateException {
+        return this.envvars.computeIfAbsent(
+            this.GH_SECRET,
+            key -> {
+                this.missingKeyExceptionThrower(key);
+                return null;
```

Comment:
@samihibrahim there is no need for return statements, please, remove them in all 3 methods

Reply:
@olenagerasimova Done but had to change the signature of the missingKeyExceptionThrower function a bit.